storage/sealer/manager.go

```diff
@@ -755,9 +761,15 @@ func (m *Manager) FinalizeReplicaUpdate(ctx context.Context, sector storiface.Se
 
 	err = multierr.Append(move(storiface.FTUpdate|storiface.FTUpdateCache), move(storiface.FTCache))
 	err = multierr.Append(err, move(storiface.FTSealed)) // Sealed separate from cache just in case ReleaseSectorKey was already called
-	// if we found unsealed files, AND have been asked to keep at least one, move unsealed
-	if moveUnsealed != storiface.FTNone && len(keepUnsealed) != 0 {
-		err = multierr.Append(err, move(moveUnsealed))
+
+	{
```

Comment:
why is there a code block here?

Reply:
Just enclosing sealedStores in a tighter scope so it's easier to tell that it's not used anywhere below; basically just a code style thing

Reply:
Why is it important that those are not used below?

Reply:
It's not important outside making it easier to see where variables are used